### PR TITLE
Feature/qa 1199 Add test reschedule l3 agent with tcpdump.

### PR DESCRIPTION
**Pull request by rpromyshlennikov**

### CODE REVIEW

File: mos_tests/neutron/python_tests/test_l3_ha.py [hunk 1]
```diff
@@ -70,6 +72,36 @@ class TestL3HA(TestBase):
     """Tests for L3 HA"""
 
     @contextmanager
+    def background_ping_from_host(self, ip_to_ping, recover_pings=50):
+        """Start ping from host to `ip_to_ping` before enter and stop it after
+
+        Return dict with ping stat
+
+        :param ip_to_ping: ip address to ping from `vm`
```
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_l3_ha.py:80]: to ping from host?

Comment (rpromyshlennikov) [inline mos_tests/neutron/python_tests/test_l3_ha.py:80]: Yep, in virtualization terms, host is machine on which hypervisor runs.

File: mos_tests/neutron/python_tests/test_l3_ha.py [hunk 2]
```diff
@@ -474,3 +504,162 @@ def test_ban_active_l3_agent_with_external_connectivity(self, router,
...
+            def start_tcpdump():
+                with node.ssh() as remote:
+                    res = remote.execute(cmd_for_tcpdump)
+                    return res
+
+            tcpdump_thread = threading.Thread(target=start_tcpdump)
+            tcpdump_thread.start()
+            return tcpdump_thread
+
+        def analyze_tcpdump(node):
```
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_l3_ha.py]: may be better call this function like get_last_icmp_packet (or what you returning here)

File: mos_tests/neutron/python_tests/test_l3_ha.py [hunk 3]
```diff
@@ -474,3 +504,162 @@ def test_ban_active_l3_agent_with_external_connectivity(self, router,
...
+        # joining threads with ssh execute tcpdump
+        for thread in tcpdump_threads:
+            thread.join(0)
+        # check that l3 active agents matching with tcpdump results
+        last_active_node = self.env.find_node_by_fqdn(active_hostname)
+        new_active_node = self.env.find_node_by_fqdn(new_active_hostname)
+        last_tcpdump_results = analyze_tcpdump(last_active_node)
+        new_tcpdump_results = analyze_tcpdump(new_active_node)
+        assert (last_tcpdump_results and new_tcpdump_results) is not None
+        assert last_tcpdump_results < new_tcpdump_results
```
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_l3_ha.py:664]: string comparison?

Comment (rpromyshlennikov) [inline mos_tests/neutron/python_tests/test_l3_ha.py:664]: Yep, 09:11:00.534495 < 09:12:00.534495 == True, am i wrong?

File: mos_tests/neutron/python_tests/test_l3_ha.py (hunk 2 above)
Comment (rpromyshlennikov) [inline mos_tests/neutron/python_tests/test_l3_ha.py]: I'm going to fix it.

File: mos_tests/neutron/python_tests/test_l3_ha.py (hunk 3 above)
Comment (rpromyshlennikov) [inline mos_tests/neutron/python_tests/test_l3_ha.py:664]: Added date to timestamp.

**Pull request merged**